spec/features/course/assessment/submission/worksheet_spec.rb

```diff
@@ -78,6 +78,28 @@
         end
       end
 
+      scenario 'I can edit my existing comment', js: true do
```

Comment:
Splitting tests might be good if it provides some guidance on where to edit (and if tests are independent). In this case the functionality all in the same area too. Maybe can compare these tests with non-js tests? 

Can use `rspec --profile -- <path>` to get the timing!